src/deploy/functions/backend.ts

```diff
@@ -80,7 +80,7 @@ export interface EventTrigger {
    * V2 will have arbitrary filters and some EventArc filters will be
    * top-level keys in the GCF API (e.g. "pubsubTopic").
    */
-  eventFilters: EventFilter[];
+  eventFilters?: EventFilter[];
```

Comment:
I don't see the code where we don't initialize `eventFilters` with an empty list - is this change really necessary?